src/utils/common/utilFunctions.ts

```diff
@@ -15,3 +16,9 @@ export function getFileExtension(fileName): string {
 export function runningInBrowser() {
     return typeof window !== 'undefined';
 }
+
+export const isFirstLogin = () => getData(LS_KEYS.IS_FIRST_LOGIN)?.status;
```

Comment:
Should this default to `true`?

Reply:
``` true```?? or ```false``` because only if a user passes through the verify page(where i set it to true)....will he be logging in for the first time